app/server/services/stats.js

```diff
@@ -104,7 +104,7 @@ function calculateStats(){
         newStats.confirmed += user.status.confirmed ? 1 : 0;
 
         // Count confirmed that are mit
-        newStats.confirmedMit += user.status.confirmed && email === "mit.edu" ? 1 : 0;
+        newStats.confirmedMit += user.status.confirmed && email === "vt.edu" ? 1 : 0;
```

Comment:
should be confirmedVT probably. doesnt super matter anywhere it seems

Reply:
Chnaged